packages/core/test/unit/sampling/service_connector.test.js

```diff
@@ -243,12 +254,38 @@ describe('ServiceConnector', function() {
 
     it('Only calls callback once after getting an invalid API response', function() {
       // Generates a client that returns an invalid (non-JSON) response
-      sandbox.stub(ServiceConnector, 'httpClient').value(generateMockClient(null, true));
+      sandbox.stub(ServiceConnector, 'httpClient').value(generateMockRulesClient(null, true));
       
       ServiceConnector.fetchTargets([], callbackObj.errCallback);
 
       callbackObj.errCallback.should.have.been.calledOnce;
     });
+
+    it('Allows LastModificationDate to be 0', function(done) {
```

Comment:
Tiny correction here. The key is "LastRuleModification".

Reply:
Good catch, thanks!